src/ctelnet.h

```diff
@@ -41,15 +41,21 @@
 #include <queue>
 #include <string>
 
-#if defined (Q_OS_MACOS) || defined(Q_OS_LINUX)
+#if defined(Q_OS_WIN32)
```

Comment:
I've put the Windows bit first so all others can be covered by an `#else` - otherwise FreedBSD would have had to be added to the existing list of non-Windows cases.